Packages/com.chisel.editor/Chisel/Editor/Editor/MaterialBrowser/ChiselEmbeddedFonts.cs

```diff
@@ -0,0 +1,50 @@
+/* * * * * * * * * * * * * * * * * * * * * *
+Chisel.Editors.ChiselEmbeddedFonts.cs
+
+License:
+Author: Daniel Cornelius
+
+* * * * * * * * * * * * * * * * * * * * * */
+
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Chisel.Editors
+{
+    public static class ChiselEmbeddedFonts
```

Comment:
I don't see where we're using this?

Reply:
Forgot to comment on this. Its been deleted as it was testing code and was no longer needed.